contrib/completion/bash/docker

```diff
@@ -1838,7 +1851,7 @@ _docker_container_start() {
 			COMPREPLY=( $( compgen -W "$options" -- "$cur" ) )
 			;;
 		*)
-			__docker_complete_containers_stopped
+			__docker_complete_containers_stopped "$@"
```

Comment:
TBH I didn't even check. It's coming from my previous PR and are leftovers of an attempt to automate certain rules on my side. It didn't work really great.